src/tests/intro.md

```diff
@@ -170,7 +170,38 @@ communicate with the server to coordinate running tests (see
 
 ## Crater
 
-TODO
+[Crater](https://github.com/rust-lang-nursery/crater) is a tool for compiling
+and running tests for _every_ crate on [crates.io](https://crates.io/). It is
```

Comment:
and also many crates from GitHub... I'm not exactly sure what the search parameters are, but I know we at least *try*.